dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/ProcessInstanceDao.java

```diff
@@ -87,4 +87,6 @@ public interface ProcessInstanceDao {
      * @return process instance
      */
     ProcessInstance queryFirstStartProcessInstance(@Param("processDefinitionCode") Long definitionCode);
+
+    ProcessInstance findSubProcessInstanceByParentId(int processInstanceId, int taskInstanceId);
```

Comment:
It's better to use `Integer`
```suggestion
    ProcessInstance findSubProcessInstanceByParentId(Integer processInstanceId, Integer taskInstanceId);
```

Reply:
Done